simple_network_sim/sampleUseOfModel.py

```diff
@@ -36,49 +29,84 @@ def main(argv):
         movementMultipliersFn=args.movement_multipliers,
     )
 
+    region, age_groups, infected = (None, None, None) \
+        if args.cmd == 'seeded' else (args.regions, args.age_groups, args.infected)
+
+    results = run_simulation(network, args.time, args.trials, args.cmd, args.input, region, age_groups, infected)
+    save_results(results, args.output_prefix, args.plot_states, args.plot_nodes)
+
+    logger.info("Took %.2fs to run the simulation.", time.time() - t0)
+
+
+def run_simulation(network, max_time, trials, mode, initialInfectionsFile, regions, age_groups, infected):
```

Comment:
I suggest we keep the cammelCase for consistency. We can change all the sytling in a single PR whenever we want.

Reply:
You're right, fixed it now, thanks